stack/.github/workflows/push-image.yml

```
@@ -54,6 +54,21 @@ jobs:
         sudo skopeo copy "oci-archive:${GITHUB_WORKSPACE}/run.oci" "docker://${DOCKERHUB_ORG}/run-${registry_repo}:${{ steps.event.outputs.tag }}"
         sudo skopeo copy "oci-archive:${GITHUB_WORKSPACE}/run.oci" "docker://${DOCKERHUB_ORG}/run-${registry_repo}:latest"
 
+        # If the repository name contains 'bionic', let's push it to legacy image locations as well:
+        #    paketobuildpacks/{build/run}:{version}-{variant}
+        #    paketobuildpacks/{build/run}:{version}-{variant}-cnb
+        if [[ ${registry_repo} == "bionic"-* ]];
+          # Strip the final part from a repo name after the `-`
+          # bionic-tiny --> tiny
+          variant=$(echo ${registry_repo} | sed 's/^[^-]*-//')
```

```suggestion
          variant="${registry_repo#bionic-}"
```
nit: Uses bash [parameter expansion](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html) to shave off the "bionic-" prefix. A little more streamlined than the sed + regex approach

---

genius!